src/uk/gov/hmcts/contino/InfraPipelineConfig.groovy

```diff
@@ -1,5 +1,5 @@
 package uk.gov.hmcts.contino
 
 class InfraPipelineConfig extends CommonPipelineConfig implements Serializable {
-
+    String expiresAfter
```

Comment:
I think this needs to be in the `CommonPipelineConfig` as you need it for both app and infra pipelines (app pipelines also do infra